packages/retail-ui/components/Autocomplete/__tests__/Autocomplete-test.tsx

```diff
@@ -156,6 +163,18 @@ describe('<Autocomplete />', () => {
     await delay(500);
     expect(wrapper.state('items')).toEqual(['1']);
   });
+
+  it(`RenderLayer is active only during focus`, () => {
+    const handleBlur = jest.fn();
+    const props = { value: '', source: [], onChange: () => '' };
+    const wrapper = mount<Autocomplete>(<Autocomplete {...props} />);
+    // @ts-ignore
+    wrapper.instance().handleBlur = handleBlur;
+
+    clickOutside();
+
+    expect(handleBlur).not.toBeCalled();
```

Comment:
Вроде все норм. 

Я бы еще предложил тест сделать более прямым и полным, в соответствии с его заголовком.
```suggestion
    expect(wrapper.find('RenderLayer').prop('active')).toBe(false);

    wrapper.find('input').simulate('focus');
    expect(wrapper.find('RenderLayer').prop('active')).toBe(true);
```

---
Выпустил тестовую версию из ветки `0.0.0-d09e1235b2`. Хорошо бы ребятам его сперва у себя потестить, прежде чем покатим в lts.

Reply:
А еще, по нашему процессу, лучше выкатить сперва в master, пусть там тоже обкатается. А ребята пока тестовую смогут использовать.

Reply:
Да, содержание теста было изначально немного иным. Название забыл поменять.

---

Те, кто запрашивал, сейчас уже на `2.13.0` сидят. И "контактное лицо" в отпуске до следующей неделе.
Тогда после ревью можно черепикнуть в мастер. И потом, после обкатки, и перед релизом `3.0`, катнуть `lts`.

Reply:
Моя мысль была скорее не про название, а про сам тест. Кажется, что предыдущее название и мой вариант теста лучше отражают суть сделанных изменений и саму проблему, потому что они менее специфичны. Но я не буду на этом настаивать.